fns-sidecars/fn-sc-processapi-report/src/main/kotlin/gov/cdc/dataexchange/model/ProcessingStatusSchema.kt

```diff
@@ -0,0 +1,36 @@
+package gov.cdc.dataexchange.model
+
+import com.google.gson.JsonObject
+import com.google.gson.annotations.SerializedName
+import java.util.*
+
+/**
+ * Defines JSON Schema for Processing Status Api Service Bus
+ * // TODO: get correct field data (except for content)
+ * @param uploadId for "upload_id" field
+ * @param destinationId for "destination_id" field
+ * @param eventType for "event_type" field
+ * @param stageName for "stage_name" field
+ * @param contentType for "content_type" field
+ * @param content for "content" field
+ */
+data class ProcessingStatusSchema(
+
+    @SerializedName("upload_id")
+    val uploadId: String = UUID.randomUUID().toString(),
+
+    @SerializedName("destination_id")
+    val destinationId: String = "dex-testing",
+
+    @SerializedName("event_type")
```

Comment:
event_type will ultimately come from our payload as well - we have a jira ticket to add that property.